recipes/cuda-cudart/meta.yaml

```
@@ -0,0 +1,143 @@
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: cuda-cudart
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart.so.{{ version }}     # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_*.dll exit 1   # [win]
+    files:
+      - lib/*.so.*                # [linux]
+      - targets/*/lib/*.so.*      # [linux]
+      - Library\bin               # [win]
+
+  - name: cuda-cudart-dev
+    run_exports:
+      weak:
+        - {{ pin_subpackage("cuda-cudart", max_pin='x') }}
+    requirements:
+      run:
+        - cuda-cudart >={{ version }}
+        - cuda-cudart-static >={{ version }}
+        - cuda-cudart-dev_{{ target_platform }} =={{ version }}  # [linux]
+    test:
+      commands:
+        - test -d $PREFIX/lib/pkgconfig                  # [linux]
+        - if not exist %LIBRARY_LIB%\cudart.lib exit 1   # [win]
+    files:
+      - lib/*.so                                # [linux]
+      - include                                 # [linux]
+      - lib/pkgconfig                           # [linux]
+      - Library\include                         # [win]
+      - Library\lib\cudadevrt.lib               # [win]
+      - Library\lib\cuda.lib                    # [win]
+      - Library\lib\cudart.lib                  # [win]
+
+  - name: cuda-cudart-dev_{{ target_platform }}
+    requirements:
+      run:
+        - cuda-cudart-static_{{ target_platform }} >={{ version }}
+    build:
+      skip: True                        # [not linux]
+      noarch: generic
+    test:
+      commands:
+        - test -L $PREFIX/targets/*/lib/libcudart.so
+    files:
+      - targets/*/lib/*.so
+      - targets/*/include/*
+      - targets/*/include/cooperative_groups/*
+
+  - name: cuda-cudart-static
+    requirements:
+      run:
+        - cuda-cudart-static_{{ target_platform }} =={{ version }}  # [linux]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcudart_static.a               # [linux]
+        - if not exist %LIBRARY_LIB%\cudart_static.lib exit 1  # [win]
+    files:
+      - lib/*.a                        # [linux]
+      - Library\lib\cudart_static.lib  # [win]
+
+  - name: cuda-cudart-static_{{ target_platform }}
+    build:
+      skip: True                        # [not linux]
+      noarch: generic
+    test:
+      commands:
+        - test -f $PREFIX/targets/*/lib/libcudart_static.a             # [linux]
+    files:
+      - targets/*/lib/*.a              # [linux]
+
+  - name: cuda-driver-dev
+    requirements:
+      run:
+        - cuda-driver-dev_{{ target_platform }} =={{ version }}  # [linux]
+    build:
+      skip: True                        # [not linux]
```

Think we should flip these

```suggestion
    build:
      skip: True                        # [not linux]
    requirements:
      run:
        - cuda-driver-dev_{{ target_platform }} =={{ version }}  # [linux]
```